webui/react/src/components/NavigationSideBar.tsx

```diff
@@ -57,6 +57,7 @@ const menuConfig = {
     { icon: 'experiment', label: 'Experiments', path: paths.experimentList() },
     { icon: 'tasks', label: 'Tasks', path: paths.taskList() },
     { icon: 'cluster', label: 'Cluster', path: paths.cluster() },
+    { icon: 'tasks', label: 'Jobs', path: paths.jobs() }, // TODO update icon
```

Comment:
icon should be available now (as `icon: 'queue'`)

Reply:
I'll need to rebase the jobqueue branch for that.. 😢

Reply:
we'll eventually use those before releasing to master.